va/va.h

```diff
@@ -1172,6 +1211,36 @@ typedef union _VAConfigAttribValContextPriority{
 #define VA_PROCESSING_RATE_ENCODE                     0x00000001
 /** \brief Driver supports decode processing rate report  */
 #define VA_PROCESSING_RATE_DECODE                     0x00000002
+
+/** \brief Attribute value for VAConfigAttribDisplayLinkHdcpWired and VAConfigAttribDisplayLinkHdcpWireless*/
+typedef union _VAConfigAttribValProtectedDisplayLink {
+    struct {
+        /** \brief session types - None = 0, Type0 = 1, Type1 = 2*/
```

Comment:
could you add comments of typex, what means of type0 , or refer HDCP document or library code

Reply:
Remove this structure since it will not be used.